src/main/java/bio/terra/app/controller/SnapshotsApiController.java

```diff
@@ -171,27 +174,30 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
 
   @Override
   public ResponseEntity<EnumerateSnapshotModel> enumerateSnapshots(
-      @Valid @RequestParam(value = "offset", required = false, defaultValue = "0") Integer offset,
```

Comment:
Can you explain why we're able to remove these annotations? Is it because they are already generated by swagger?

Reply:
Yes, they're in the swagger generated code. In some of our controller subclasses we've copied these annotations down. The main downside to doing this is that now you've duplicated generated code, and our copy doesn't change when the generated code copy does. The other downside is it makes the code harder to read. Also it doesn't provide any benefit :)